modules/terraform/plan.go

```diff
@@ -7,23 +7,55 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-// InitAndPlan runs terraform init and plan with the given options and return stdout/stderr from the apply command.
-func InitAndPlan(t *testing.T, options *Options) int {
-	exitCode, err := InitAndPlanE(t, options)
+// InitAndPlan runs terraform init and plan with the given options and returns stdout/stderr from the plan command.
```

Comment:
```suggestion
// InitAndPlan runs terraform init and plan with the given options and returns stdout/stderr from the plan command.
// This will fail the test if there is an error in the command.
```

I know this wasn't in the original comment text, but we started to mention that the non-E versions fail the test.

Reply:
okay, added these lines to comments, waiting for the build to complete, then will be ready to force-push